contractcourt/channel_arbitrator.go

```diff
@@ -1209,13 +1207,12 @@ func (c *ChannelArbitrator) checkChainActions(height uint32,
 	// either learn of it eventually from the outgoing HTLC, or the sender
 	// will timeout the HTLC.
 	for _, htlc := range c.activeHTLCs.incomingHTLCs {
-		payHash := htlc.RHash
-
 		// If we have the pre-image, then we should go on-chain to
 		// redeem the HTLC immediately.
-		if _, ok := c.cfg.PreimageDB.LookupPreimage(payHash[:]); ok {
+		if _, ok := c.cfg.PreimageDB.LookupPreimage(htlc.RHash); ok {
 			log.Tracef("ChannelArbitrator(%v): preimage for "+
-				"htlc=%x is known!", c.cfg.ChanPoint, payHash[:])
+				"htlc=%x is known!", c.cfg.ChanPoint,
+				htlc.RHash[:])
```

Comment:
I think the `Hash` type has a `String` method, so you can use "%v" to print

Reply:
`htlc.RHash` is still a `[32]byte`, still need `%x`

Reply:
Ah, but it can still be used when calling `LookupPreimage(payhash lntypes.Hash)`? Did not know that :)

Reply:
yep, since `lntypes.Preimage` is a named type and `[32]byte` is an unnamed type, they can be casted implicitly bc the underlying raw type is the same